internal/kgateway/extensions2/plugins/trafficpolicy/jwt.go

```diff
@@ -158,7 +162,14 @@ func ProviderName(resourceName, providerName string) string {
 	return fmt.Sprintf("%s_%s", resourceName, providerName)
 }
 
-func translateProvider(krtctx krt.HandlerContext, provider v1alpha1.JWTProvider, policyNs string, configMaps krt.Collection[*corev1.ConfigMap]) (*jwtauthnv3.JwtProvider, error) {
+func translateProvider(
+	krtctx krt.HandlerContext,
+	provider v1alpha1.JWTProvider,
+	policyNs string,
+	configMaps krt.Collection[*corev1.ConfigMap],
+	resolver backendResolver,
+	policySrc ir.ObjectSource,
```

Comment:
this is not the policy right?


```suggestion
	gwExtSrc ir.ObjectSource,
```